pipeline.py

```diff
@@ -115,7 +115,7 @@ def pipeline(
         sdg_scale_factor: int = 2,  # Renamed upstream https://github.com/instructlab/instructlab/blob/f7d40f6ed5112d59132dd832bd332fa6fbbe7010/src/instructlab/configuration.py#L279-L290
         sdg_pipeline: str = SDG_PIPELINE,
         sdg_max_batch_len: int = MAX_BATCH_LEN,
-        sdg_sample_size: float = 1.0,
+        sdg_sample_size: float = None,
```

Comment:
This is problematic when launching through RHOAI UI:
![image](https://github.com/user-attachments/assets/8b2e7f7a-60b7-4abb-a1b5-90c86352691d)

You can't submit the workflow without actually setting the value. This is related to following bug:
https://issues.redhat.com/browse/RHOAIRFE-467
https://github.com/kubeflow/pipelines/issues/11331

Reply:
Fixed, thanks for pointing it out.